<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-8TP6OQIJ" failures="1" tests="2" name="Test" time="21.219" errors="0" timestamp="2023-05-06T20:07:31 IST">
  <testcase classname="TestCases.VerifyLogin" name="invalidLogin" time="0.646">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	error message is not valid expected [Please check your username. If you still can&amp;apos;t log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
error message is not valid expected [Please check your username. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at Rsources.CommonMethods.handleAssertions(CommonMethods.java:19)
at TestCases.VerifyLogin.invalidLogin(VerifyLogin.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- invalidLogin -->
  <testcase classname="TestCases.VerifySignup" name="signup" time="6.738"/>
  <testcase classname="Rsources.baseClass" name="@BeforeTest ExtentReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest ExtentReport -->
  <testcase classname="Rsources.baseClass" name="@AfterTest endReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest endReport -->
</testsuite> <!-- Test -->
